content/cpp/concepts/vectors/terms/empty.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.empty()'
+Description: 'Returns a boolean representing whether the vector is empty (i.e. whether its size is 0)'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Vectors'
+  - 'Arrays'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`.empty()`** method returns `true` if the size of the vector is 0. Otherwise, if it has at least one element, returns `false`.
```

Comment:
```suggestion
The **`.empty()`** method returns `true` if the size of the vector is 0. Otherwise, if it has at least one element, it returns `false`. Besides this, it is used to check if the vector contains any elements. It is mostly used in conditional statements.
```
I have added some detailed information about the empty method of vector, You can also add some more information if needed.